pyof/v0x01/controller2switch/packet_out.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, xid=None, buffer_id=NO_BUFFER, in_port=Port.OFPP_NONE,
         self.buffer_id = buffer_id
         self.in_port = in_port
         self.actions = [] if actions is None else actions
-        self.data = data
+        self.data = BinaryData() if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.